Dockerfile

```diff
@@ -19,6 +22,7 @@ ENV HEAP_LOG_PATH /usr/share/service/dump
 ENV JAVA_OPS -Xms256m -Xmx512m -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=$HEAP_LOG_PATH
 ENV JAVA_DEBUG_OPS -verbose:gc -XX:+PrintGCDetails -XX:+PrintGCTimeStamps -Xloggc:$HEAP_LOG_PATH/garbage-collection.log
 ENV DEBUG_MODE false
+ENV KEYCLOAK_SERVER_REALM=$KEYCLOAK_SERVER_REALM
```

Comment:
We don't need to receive the realm value from the build arg, we can set this variable with a dummy value and change it during execution, or even better, move it to a config file itself.

Reply:
Yup. That part is gone.